packages/shared/src/hooks/useLogin.ts

```diff
@@ -55,18 +51,13 @@ const useLogin = ({
   onSuccessfulLogin,
   queryEnabled = true,
   queryParams = {},
-  enableSessionVerification = false,
+  session,
```

Comment:
Also, I am actually in a dilemma with how we use react query. Do we pass this value as a prop or do we fetch this through `client.getQueryData`? Thoughts?